pin multipy

PaliC opened the pull request:
Pins multipy to prevent breakages in the torch CI due to multipy changes.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #90942

PaliC committed:
pin multipy

[ghstack-poisoned]

PaliC committed:
Update on "pin multipy"

Pins multipy to prevent breakages in the torch CI due to multipy changes.




[ghstack-poisoned]

huydhn reviewed (approved):
Thank you for fixing the issue and keep trunk green!

PaliC committed:
Update on "pin multipy"

Pins multipy to prevent breakages in the torch CI due to multipy changes.




[ghstack-poisoned]

PaliC commented:
@pytorchbot merge -g

PaliC committed:
Update on "pin multipy"

Pins multipy to prevent breakages in the torch CI due to multipy changes.




[ghstack-poisoned]

PaliC commented:
@pytorchbot merge -g
